shell/browser/electron_net_log.cc

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
```

Comment:
Why was this removed by Chrome? Can we get away with removing it too instead of copying over the logic?

Reply:
nvm this was removed :)